internal/rbd/driver/driver.go

```diff
@@ -33,12 +33,10 @@ import (
 
 // Driver contains the default identity,node and controller struct.
 type Driver struct {
-	cd *csicommon.CSIDriver
-
+	cd  *csicommon.CSIDriver
```

Comment:
unnecessary change.

Reply:
The formatting seems to have been incorrect, this is how it should be, I think?